### ISSUE TITLE
Adding padding or margins

### ISSUE THREAD

**Issue by Benjamin72070** [ChurchApps/FreeShow#297]
I have tried adding both padding and margin css in both the textbox and item custom css.  Neither have the effect of keeping the text inside a box from butting right up against the edge.  I have attached an image to demonstrate what I mean,  I have circled in blue the problem.  The red circle is where I can press enter for a line break to keep it from happening, but this causes a problem when setting a template to override the style for another output because the text keeps the formatting and then breaks at the same point even if it does not need to.  

![Freeshow1](https://github.com/ChurchApps/FreeShow/assets/145783321/a6b57629-15fa-40e1-8747-573f9afb206e)

I have just realized as writing this that I can redesign the template with the text above a textbox to create the the effect and shrink the top layer by however many pixels I need.  it would still be nice to be able to use padding or margins without breaking the design.

**Comment by vassbo** [comment]
Maybe an option to change the padding, but you should be able to add this value to the "Item" CSS, and it should work: "padding: 30px;"

**Comment by Benjamin72070** [comment]
I put padding in the item field and it broke the template.  I just tried the textbox over a textbox for a scripture template.  Sometimes it puts the text in the top textbox.  Sometimes it puts it in the bottom textbox layer.

**Comment by vassbo** [comment]
You have to select only one item at a time, so the style don't get applied to both. Also the order of the items determines where the text is put.

**Comment by Benjamin72070** [comment]
Yeah, I just figured that out.  I changed the order in the "Items".  It at first put the metadata where the verse text was supposed to go, then I clicked on the template a couple of times to rotate what textbox the metadata and the verse were placed in.  I do not know if that is intended or not.  I suspect it works that way because the coding to add multiple verses to a slide required it.  I am not sure that my template with the "nested" textboxes would work if I changed the scripture preferences to more than one verse per slide.
